### ISSUE TITLE
unable to load /settings/things

### ISSUE THREAD

**Issue by ccutrer** [openhab/openhab-core#3316]
I'm running 3.4.0. A thing is offline for reasons unknown. When I try to go to `/settings/things` in the browser, the AJAX call to `/rest/things` gives a 200, but has malformed output. If I look in openhab.log, I see

```
2023-01-13 10:35:45.204 [ERROR] [rg.apache.cxf.jaxrs.utils.JAXRSUtils] - Problem with writing the data, class org.openhab.core.io.rest.Stream2JSONInputStream, ContentType: application/json
2023-01-13 10:35:45.204 [ERROR] [internal.JSONResponseExceptionMapper] - Unexpected exception occurred while processing REST request.
java.lang.IllegalArgumentException: can't parse argument number: 
        at java.text.MessageFormat.makeFormat(MessageFormat.java:1454) ~[?:?]
        at java.text.MessageFormat.applyPattern(MessageFormat.java:492) ~[?:?]
        at java.text.MessageFormat.<init>(MessageFormat.java:371) ~[?:?]
        at java.text.MessageFormat.format(MessageFormat.java:860) ~[?:?]
        at org.openhab.core.internal.i18n.I18nProviderImpl.getText(I18nProviderImpl.java:349) ~[?:?]
        at org.openhab.core.thing.i18n.ThingStatusInfoI18nLocalizationService.getTranslationForClass(ThingStatusInfoI18nLocalizationService.java:147) ~[?:?]
        at org.openhab.core.thing.i18n.ThingStatusInfoI18nLocalizationService.translateDescription(ThingStatusInfoI18nLocalizationService.java:128) ~[?:?]
        at org.openhab.core.thing.i18n.ThingStatusInfoI18nLocalizationService.getLocalizedThingStatusInfo(ThingStatusInfoI18nLocalizationService.java:86) ~[?:?]
        at org.openhab.core.io.rest.core.internal.thing.ThingResource.convertToEnrichedThingDTO(ThingResource.java:743) ~[?:?]
        at org.openhab.core.io.rest.core.internal.thing.ThingResource.lambda$0(ThingResource.java:299) ~[?:?]
...
Caused by: java.lang.NumberFormatException: For input string: ""
        at java.lang.NumberFormatException.forInputString(NumberFormatException.java:67) ~[?:?]
        at java.lang.Integer.parseInt(Integer.java:678) ~[?:?]
        at java.lang.Integer.parseInt(Integer.java:786) ~[?:?]
        at java.text.MessageFormat.makeFormat(MessageFormat.java:1452) ~[?:?]
        ... 74 more
```

The JSON the browser gets ends with:

```
...
{"statusInfo":{"status":"ONLINE","statusDetail":"NONE"},"editable":false,"label":"Master Toilet Room Occupancy","bridgeUID":"mqtt:broker:hiome","UID":"mqt"{\"error\":{\"message\":\"can\\u0027t parse argument number: \",\"http-code\":500,\"exception\":{\"class\":\"java.lang.IllegalArgumentException\",\"message\":\"can\\u0027t parse argument number: \",\"localized-message\":\"can\\u0027t parse argument number: \",\"cause\":\"java.lang.NumberFormatException\"}}}"
```

If I do `things list` in console, the list ends with:

```
mqtt:topic:1576466893 (Type=Thing, Status=ONLINE, Label=Suzie's Room Occupancy, Bridge=mqtt:broker:hiome)
An unexpected error occurred during execution.
```

and I see the following similar error in my log:

```
2023-01-13 10:37:45.813 [ERROR] [b.core.io.console.ConsoleInterpreter] - An error occurred while executing the console command.
java.lang.IllegalArgumentException: can't parse argument number: 
        at java.text.MessageFormat.makeFormat(MessageFormat.java:1454) ~[?:?]
        at java.text.MessageFormat.applyPattern(MessageFormat.java:492) ~[?:?]
        at java.text.MessageFormat.<init>(MessageFormat.java:371) ~[?:?]
        at java.text.MessageFormat.format(MessageFormat.java:860) ~[?:?]
        at org.openhab.core.internal.i18n.I18nProviderImpl.getText(I18nProviderImpl.java:349) ~[?:?]
        at org.openhab.core.thing.i18n.ThingStatusInfoI18nLocalizationService.getTranslationForClass(ThingStatusInfoI18nLocalizationService.java:147) ~[?:?]
        at org.openhab.core.thing.i18n.ThingStatusInfoI18nLocalizationService.translateDescription(ThingStatusInfoI18nLocalizationService.java:128) ~[?:?]
        at org.openhab.core.thing.i18n.ThingStatusInfoI18nLocalizationService.getLocalizedThingStatusInfo(ThingStatusInfoI18nLocalizationService.java:86) ~[?:?]
        at org.openhab.core.thing.internal.console.ThingConsoleCommandExtension.printThings(ThingConsoleCommandExtension.java:184) ~[?:?]
        at org.openhab.core.thing.internal.console.ThingConsoleCommandExtension.execute(ThingConsoleCommandExtension.java:89) ~[?:?]
        at org.openhab.core.io.console.ConsoleInterpreter.execute(ConsoleInterpreter.java:57) [bundleFile:?]
        at org.openhab.core.io.console.karaf.internal.CommandWrapper.execute(CommandWrapper.java:79) [bundleFile:?]
        at org.apache.karaf.shell.impl.console.osgi.secured.SecuredCommand.execute(SecuredCommand.java:68) [bundleFile:4.3.7]
        at org.apache.karaf.shell.impl.console.osgi.secured.SecuredCommand.execute(SecuredCommand.java:86) [bundleFile:4.3.7]
        at org.apache.felix.gogo.runtime.Closure.executeCmd(Closure.java:599) [bundleFile:4.3.7]
        at org.apache.felix.gogo.runtime.Closure.executeStatement(Closure.java:526) [bundleFile:4.3.7]
        at org.apache.felix.gogo.runtime.Closure.execute(Closure.java:415) [bundleFile:4.3.7]
        at org.apache.felix.gogo.runtime.Pipe.doCall(Pipe.java:416) [bundleFile:4.3.7]
        at org.apache.felix.gogo.runtime.Pipe.call(Pipe.java:229) [bundleFile:4.3.7]
        at org.apache.felix.gogo.runtime.Pipe.call(Pipe.java:59) [bundleFile:4.3.7]
        at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
        at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.NumberFormatException: For input string: ""
        at java.lang.NumberFormatException.forInputString(NumberFormatException.java:67) ~[?:?]
        at java.lang.Integer.parseInt(Integer.java:678) ~[?:?]
        at java.lang.Integer.parseInt(Integer.java:786) ~[?:?]
        at java.text.MessageFormat.makeFormat(MessageFormat.java:1452) ~[?:?]
        ... 23 more
```

I've tracked this down to a `bondhome:bondBridge` thing type with a thing status of `bondhome:bondBridge:BD30179
OFFLINE (COMMUNICATION_ERROR): @text/offline.comm-error.api-call-failed` so I'll be fixing the i18n there, but even so, core should be tolerant of a bad binding and fall back to a non-localized string.

**Comment by ccutrer** [comment]
Looks like https://github.com/openhab/openhab-addons/pull/14103 already fixed the binding

**Comment by lolodomo** [comment]
Maybe the fix should be back ported to 3.4.x.

**Comment by ccutrer** [comment]
Thanks for the fix!
